addons/binding/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/internal/NukiHandlerFactory.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.nuki.internal;
+
+import java.util.Set;
+
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.openhab.binding.nuki.NukiBindingConstants;
+import org.openhab.binding.nuki.handler.NukiBridgeHandler;
+import org.openhab.binding.nuki.handler.NukiSmartLockHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.Sets;
+
+/**
+ * The {@link NukiHandlerFactory} is responsible for creating things and thing
+ * handlers.
+ *
+ * @author Markus Katter - Initial contribution
+ */
+public class NukiHandlerFactory extends BaseThingHandlerFactory {
+
+    private final Logger logger = LoggerFactory.getLogger(NukiHandlerFactory.class);
+    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Sets
+            .union(NukiBindingConstants.THING_TYPE_BRIDGE_UIDS, NukiBindingConstants.THING_TYPE_SMARTLOCK_UIDS);
+
+    @Override
+    public boolean supportsThingType(ThingTypeUID thingTypeUID) {
+        boolean isSupported = SUPPORTED_THING_TYPES_UIDS.contains(thingTypeUID);
+        logger.trace("NukiHandlerFactory:supportsThingType({}) - isSupported[{}]", thingTypeUID, isSupported);
+        if (!isSupported) {
+            logger.warn("ThingType[{}] is not supported! Please check your .things configuration!", thingTypeUID);
```

Comment:
Can you please reduce the log level to trace, because by default every thing in the thing file gets passed to this method and this creates a huge amount of log output. Maybe even remove the log output completely.